### DISCUSSION TITLE
Seamless update management of headless cms

### DISCUSSION THREAD

**Discussion by firozemzr** [OSBDAlliance/headlesscms#8] [Ideas]
Headless CMS will continuously be improved and released. The projects that will be using headless cms should be able to upgrade or fallback to earlier version easily without breaking their project or losing data.

**Comment by mohammad-arif-rahim** [comment]
May not be wise. What if a developer changes some of the deployment codes and we force update the CMS to next version? We should provide an option to update the CMS to the developer, it is his choice/risk if he wants to do it or not.

**Reply by firozemzr** [reply]
I think this is quite critical for overall lifecycle management. If any developer changes the core code, then he has to manage the upgrade manually.

But automatic upgrade features like wordpress or other cms is very much necessary. It makes life very easy and avoids a lot of house keeping everytime there is a new release.

And initially there will be many releases.
